brokerbot.go

```diff
@@ -120,95 +122,89 @@ func initTokens() {
 }
 
 func handleDefaultPort(w http.ResponseWriter, r *http.Request) {
-	log.Println("Heartbeat")
-	timeSinceLastHeartbeat = time.Now()
-	fmt.Fprintln(w, "Hello World!")
+	fmt.Fprintln(w, "OK")
 }
 
 func handleMessage(s *discordgo.Session, m *discordgo.MessageCreate) {
-	log.Printf("time since last heartbeat: %s", time.Since(timeSinceLastHeartbeat))
-	/* Validation */
 	if m.Author.ID == s.State.User.ID {
-		// Prevent the bot from talking to itself.
+		// Ignore messages from self.
 		return
 	}
 
-	isBotMention, msg := hasBotMention(s, m)
+	splitMsg := strings.Fields(m.ContentWithMentionsReplaced())
```

Comment:
Why do we prefer to do this, instead of relying on the Mentions mechanism within the discordgo library like before?

Reply:
This eliminates the messy string parsing you were doing previously to remove the mentions. Now it's easier with `messagelib.RemoveMentions`.